clusterloader2/testing/load/config.yaml

```diff
@@ -269,9 +270,15 @@ steps:
     - basename: medium-deployment
       objectTemplatePath: networkpolicy.yaml
     {{end}}
+    {{if $ENABLE_DNSTESTS}}
+    - basename: medium-deployment
+      objectTemplatePath: dnsperf-deployment.yaml
+    {{else}}
     - basename: medium-deployment
       objectTemplatePath: deployment.yaml
+    {{end}}
       templateFillMap:
+        ENABLE_DNSTESTS: {{$ENABLE_DNSTESTS}}
```

Comment:
This templateFillMap section is for 'deployment.yaml' or 'dnsperf-deployment.yaml' file.

Please move it from here to newly created templateFillMap section in:
" - basename: medium-deployment
     objectTemplatePath: configmap.yaml
     templateFillMap:
        EnableDNSTests: {{$ENABLE_DNSTESTS}}"

Also please try to be consistent with the naming style in templateFillMaps (so CamelCase).

Reply:
Thanks for catching this, will fix.

Reply:
Also, would I need to fill this in everytime for configmap, or only when I set it to true?

Reply:
Not sure if I understand the question, but you need to set EnableDNSTests field in templateFillMap when configmap.yaml will read this field (so basically all the time).